index.html

```diff
@@ -14,24 +14,24 @@
         <table>
             <tr>
                 <th>
-                    <div class="smile" href="">😀</div>
+                    <div class="smile">😀</div>
```

Comment:
You need to attach a listener to a parent element, then check `event.target`

```
parentElement.addEventListener('click', () => {
  if (event.target.classList.contains("smile")) {
    // then change counter text 
  }
})
```

According to your markup parent element, you need to use the `tr`

Reply:
I try rewrite,  but it is doesn't`t work now, could you help me to find mistakes please?

I add a class voting to tr element and then attached listener

voting.addEventListener("click", (e) => {
  if (e.target.classList.contain('smile')) {
    const counter = e.target.nextElementSibling;
    let count = counter.textContent;
    count++;
    counter.innerText = count;
}
})